CMakeLists.txt

```diff
@@ -39,27 +39,27 @@ if("${VCLIBS_TARGET_ARCHITECTURE}" MATCHES "^[xX]86$")
     # Note that we set _WIN32_WINNT to a high level to make declarations available, but still engage downlevel
     # runtime dynamic linking by setting our own _STL_WIN32_WINNT back to Windows XP.
     add_compile_definitions(_X86_ _VCRT_WIN32_WINNT=0x0501 _STL_WIN32_WINNT=0x0501
-        _WIN32_WINNT=0x0602 NTDDI_VERSION=NTDDI_WIN8)
+        _WIN32_WINNT=0x0A00 NTDDI_VERSION=NTDDI_WIN10_CO)

```

Comment:
All architectures set `_WIN32_WINNT` and `NTDDI_VERSION` to the same values - should we add these outside of the branching structure for DRY? (i.e., on line 70 where there's currently an extraneous trailing space in the compile definitions)

Reply:
Yep, this works! Thanks, I didn't think about extracting duplication.